static/js/rendered_markdown.js

```diff
@@ -161,8 +162,14 @@ export const update_elements = (content) => {
         if (isValid(timestamp)) {
             const text = $(this).text();
             const rendered_time = timerender.render_markdown_timestamp(timestamp, text);
+            // Update the value of timestamp text to pass the tests.
             $(this).text(rendered_time.text);
```

Comment:
The only reason to keep this line is, because it updates the value of timestamp text which is used [here](https://github.com/zulip/zulip/blob/master/frontend_tests/node_tests/rendered_markdown.js#L184) for testing.

Reply:
We could probably just make the test check `.html()` instead, right?

Reply:
Sorry, I didn't understand your question, Can you elaborate your question a little more?

Reply:
@nikhilmaske-2001 He means that you can change the test accordingly to the new way you are using it. So, removing the older way in the actual code and updating the tests.

Reply:
Thanks @Signior-X for the explanation.